LayoutTests/imported/w3c/web-platform-tests/webstorage/symbol-props.window-expected.txt

```diff
@@ -2,14 +2,14 @@
 PASS localStorage: plain set + get (loose)
 PASS localStorage: plain set + get (strict)
 PASS localStorage: defineProperty + get
-PASS localStorage: defineProperty not configurable
+FAIL localStorage: defineProperty not configurable assert_true: configurable expected true got false
```

Comment:
The two subtests that are newly failing in this test are due to me dropping the `setConfigurable(true)` in the generated `defineOwnProperty()` functions. Note that both Firefox and Chrome are failing these subtests in the same way so we are actually aligning with other browser engines here. Keeping the `setConfigurable(true)` is causing other WPT tests to fail, which are passing in other browsers.

My plan is to get this test updated upstream to agree with browser engines.